packages/mnemonic-seed-manager/package.json

```diff
@@ -15,17 +15,19 @@
   },
   "dependencies": {
     "@veramo/core": "^3.1.0",
```

Comment:
Since develop currently is using all Veramo packages at 3.1.1-next15 because we need the json-ld infra created there, it makes sense to update all the @ veramo deps to the same version, so we are sure it still works once merged

Reply:
Fixed!